operator/pkg/operator.go

```diff
@@ -475,7 +490,13 @@ func (o *Operator) afterHandlingBatchV3(log *servicemanager.ContractAlignedLayer
 	}
 }
 
-func (o *Operator) verify(verificationData VerificationData, results chan bool) {
+func (o *Operator) verify(verificationData VerificationData, disabledVerifiersBitmap *big.Int, results chan bool) {
+	IsVerifierDisabled := IsVerifierDisabled(disabledVerifiersBitmap, verificationData.ProvingSystemId)
+	if IsVerifierDisabled {
+		o.Logger.Infof("Verifier %s is disbled. Skipping verifie", verificationData.ProvingSystemId.String())
```

Comment:
```suggestion
		o.Logger.Infof("Verifier %s is disbled. Returning false", verificationData.ProvingSystemId.String())
```
Just to make it a little more explicit